README.md

```diff
@@ -37,6 +37,10 @@ A simple extension that chooses and applies a *random* theme.
 
 - `randomThemeSwitcher.themeList`: a list of string with labels of themes.
 
+- `oneMoreTimeThemeList`: a list of themes that will be kept one more time before being changed when randomized
```

Comment:
I think that like this sound better, we don't need to "expose" the internal of how the extension work. Also like this is more clear

```suggestion
- `preventReloadThemeList`: some themes, after they are applied, reload the instance. If are present in this list the extension will not set a new random theme after they are applied.
```

Reply:
Hey @zanza00 !
Thanks for your replies:

I didn't understand what the problem was with the list of materials,
I thought you had preferences about that suite.😅
Now it's clear and I agree with you on the name.